src/api/configureApi.ts

```diff
@@ -26,11 +26,23 @@ import { createParticipantsRouter } from './participantsRouter';
 import { createUsersRouter } from './usersRouter';
 
 const BASE_REQUEST_PATH = '/api';
-function bypassHandlerForPaths(middleware: express.Handler, ...paths: string[]) {
+const BYPASS_PATHS = [
+  `/favicon.ico`,
+  `${BASE_REQUEST_PATH}/`,
+  `${BASE_REQUEST_PATH}/metrics`,
+  `${BASE_REQUEST_PATH}/health`,
+  `${BASE_REQUEST_PATH}/keycloak-config`,
+];
+
+function bypassHandlerForPaths(middleware: express.Handler, ...paths: (string | string[])[]) {
```

Comment:
Instead of using a string array for [path, method], it would be better to use an object. `path.url` and `path.method` looks better than `path[0]` and `path[1]`.